image/Dockerfile

```diff
@@ -38,13 +38,10 @@ RUN python3 -m venv /opt/venv
 ENV PATH=/opt/venv/bin:$PATH
 RUN pip install --upgrade pip
 COPY golem_blender_app/requirements.txt /tmp
+# Use following lines to work with the unreleased task-api module in requirements.txt
```

Comment:
fwiw This approach is inferior to the previous one because anytime you update the unreleased `task-api` package you invalidate the cache of all the dependencies. In the previous approach you only reinstall `task-api`.

So please do not commit these changes.

Reply:
i had some issues with installing the package inside the docker container, this is why i made this change.

Will try to revert and see if the setup.py changes fixed those issues

Reply:
still works, fixed :D